modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+
+_*A convergence window is a period during which multiple actors, such as users, applications, or clusters, can modify the same document in an active-active topology, before the changes have fully converged across all clusters._
+
+== XDCR Conflict Logger Overview
+
+* Conflict logging is asynchronous and done on a best-effort basis, meaning, not all conflicts may be logged.
+
+* Conflicts are expected to be rare during XDCR replication.
+If there is an upsurge of conflicts, so that logging conflicts may cause replication issues, XDCR stops logging conflicts temporarily to prioritize replication.
+
+* If any error occurs, due to resource issues or a delay in writing to the conflict log collection, XDCR may pause or stop the logging process, report the error type for the pipeline, and continue with replication so that there is no High Availability issue.
+
+* xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc[Conflict Resolution] process remains unchanged.
+
+IMPORTANT: Even if conflict logging fails, XDCR continues with conflict resolution to make sure of High Availability.
+Conflict resolution is not changed by conflict logging.
+Conflict logging is informational only.
+
+[#using-of-xdcr-conflict-logging]
+== Using the XDCR Conflict Logging Feature
+
+To log conflicts during an XDCR, follow these steps:
+
+. As a prerequisite, to enable the conflict logging feature, xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[Enable the Cross Cluster Versioning] setting on each bucket of the XDCR topology.
+Do this by setting the bucket property `enableCrossClusterVersioning` to `true` on all the buckets in the XDCR topology clusters.
+As a result, additional metadata called, xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#hlv-data-maintained-in-xattr[Hybrid Logical Vector (HLV)], also known as version vector, is added to the documents and updated as the documents replicate.
++
+To enable the bucket property `enableCrossClusterVersioning`:
+
+* Using REST API, see xref:rest-api:rest-bucket-create.adoc#example-enablecrossclusterversioning-edit[Example: Turning on enableCrossClusterVersioning, when Editing].
+
+* From the UI, see xref:manage:manage-buckets/edit-bucket.adoc#edit-bucket-for-eccv[Edit a Bucket to Enable Cross Cluster Versioning].
++
+[NOTE]
+====
+* Once `enableCrossClusterVersioning` is enabled, it cannot be disabled.
+As a result, HLV also cannot be disabled.
+
+* The conflict logging feature detects and logs conflicts only when the property `enableCrossClusterVersioning` is set to `true` for all the buckets in the replication topology.
+Make sure you set `enableCrossClusterVersioning` to `true` in the bucket properties when adding new buckets or new clusters to the existing replication topology.
+====
+
+. When creating the XDCR replication, configure and enable conflict logging.
```

Comment:
Should say "creating or updating" so that people are aware that they can configure conflict logging for a replication that already exists.

. When creating or updating the XDCR replication,